0075-mrna/sbatch.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+. ../common.sh
+
+task=$1
+log=$logDir/sbatch.log
+
+# NOTE!! The following must have the identical value set and used in mrna.sh
+out=$task.done
```

Comment:
Ditto.

Reply:
Done. Also adjusted the use of $out and $outUncompressed in mrna.sh